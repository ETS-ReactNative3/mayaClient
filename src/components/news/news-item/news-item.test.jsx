import NewsItem from './news-item';

describe('NewsItem', () => {
	it('should render correctly', () => {
		const wrapper = shallow(
			<NewsItem key={1} news={
				{
					"title": "titleLol",
					"text": "textLol"
				}
			} />
		)
		expect(wrapper).toMatchSnapshot();
	})
});

