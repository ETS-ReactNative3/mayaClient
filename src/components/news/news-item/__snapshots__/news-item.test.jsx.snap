// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsItem should render correctly 1`] = `
<div>
  <h3>
    <b>
      title: 
    </b>
    titleLol
  </h3>
  <h5>
    <i>
      Text: 
    </i>
    textLol
  </h5>
</div>
`;
